main.tf

```diff
@@ -592,3 +595,54 @@ resource "aws_config_config_rule" "iam_mfa_for_console_access" {
     "aws_lambda_permission.iam_mfa_for_console_access",
   ]
 }
+
+### RESTRICTED COMMON PORTS: ACCESS
+resource "aws_config_config_rule" "restricted_common_ports_access" {
+  count = "${var.create_config_rules && ! local.exclude_restricted_common_ports_access ? 1 : 0}"
+
+  name             = "restricted-common-ports-access"
+  description      = "Checks whether security groups that are in use disallow unrestricted incoming TCP traffic to the specified ports."
+  input_parameters = "{\"blockedPort1\":\"22\",\"blockedPort2\":\"3389\"}"
```

Comment:
Is there reason for the escapes? Terraform can handle maps?

Reply:
reason: that's the way the value came back when importing an existing resource.

Reply:
How is this rule different from restriced_ssh and restricted_incoming traffic which looks to already address port 22 and 3389/

Reply:
Ran into the same thing before. The argument on the resource accepts strings, not objects. But, you could write it as a map, and use the `jsonencode` function...

* https://www.terraform.io/docs/providers/aws/r/config_config_rule.html#input_parameters

Reply:
restricted ssh only covers SSH. cheaper to have one rule for SSH and RDP. if this approach is good, we'll delete restricted_ssh.

restricted-incoming-traffic is what this both of these new rules are based on (as far as the underlying managed rule). Traffic is vague, and we're limited to five blocked ports in the console so I figured we'd be specific about the intent...default database ports. "restricted-incoming-traffic" can also be deleted if this is accepted.

Two different rules (remote access and database) also gives flexibility for the responding event pipeline.

Reply:
I was referring to the resource title "restricted_incoming_traffic" which addresses port 3389. With your addition there would be 3 rules all looking to do the same thing.